### ISSUE TITLE
Question on Reproducing MCNet Performance on SensatUrban & Request for Pretrained Model

### ISSUE THREAD

**Issue by Ker-2311** [ranhaogong/MCNet#2]
Hello,

First, thank you for open-sourcing the MCNet code. I recently tried to reproduce the results reported in the paper on the SensatUrban dataset. I followed the instructions in the repository to set up the environment, prepare the data, and run the training pipeline.

However, the performance I achieved (e.g., mIoU on the validation set) falls noticeably short of the numbers reported in the paper, and I haven’t been able to reach the stated performance level. (In the End, I will attach my training result)

To help pinpoint the issue, I’ve already checked the following aspects without finding a clear cause:

Data Preparation: Confirmed that the SensatUrban dataset was downloaded, extracted correctly, and the train/validation split was processed as required.

Hyperparameters: Largely followed the default configuration provided in the repository or the parameters suggested in the paper (e.g., learning rate, batch size, number of iterations).

Environment: Tried to match the recommended PyTorch version, CUDA version, and main Python packages.

There might be subtle implementation details, differences in data preprocessing, or hyperparameter sensitivities that are hard to notice but cause the performance gap.

Therefore, I’d like to make requests, if possible:

**Could you please provide the pretrained model weights​? This would greatly help the community in reproducing, verifying, and building upon the work.**

Thank you very much for your work and time! I look forward to your response.

<img width="920" height="390" alt="Image" src="https://github.com/user-attachments/assets/4d2b8c71-2444-4fb8-8c0b-1d95f463b72a" />

<img width="1296" height="395" alt="Image" src="https://github.com/user-attachments/assets/31feab05-a127-4734-873a-7e8534d5166b" />

<img width="2129" height="1124" alt="Image" src="https://github.com/user-attachments/assets/62c35941-e593-4e5a-84c8-14ea2d99db9a" />

<img width="2094" height="545" alt="Image" src="https://github.com/user-attachments/assets/c2b1736c-7fc0-4ebd-9009-4c5ea3471ca1" />

**Comment by Ker-2311** [comment]
@ranhaogong
